presto-main/src/main/java/com/facebook/presto/sql/planner/PlanFragment.java

```diff
@@ -139,13 +140,13 @@ public boolean isOutputTableWriterFragment()
         return outputTableWriterFragment;
     }
 
-    @JsonProperty
+    @JsonIgnore
```

Comment:
you'll need to make the statsAndCost field optional or it will fail the `requireNonNull()` check after json deserialization.

Reply:
Yeah. The Travis failure is legit.